### DISCUSSION TITLE
Development Guide

### DISCUSSION THREAD

**Discussion by kleenkanteen** [lukevella/rallly#760] [General]
In the developement guide in the readme, shouldn't it say to write `yarn install` before `yarn dev`?

![image](https://github.com/lukevella/rallly/assets/67878058/d54b3bb5-d877-438c-b82d-d70a5e1da69c)

Ok to make a PR to add it?

**Comment by lukevella** [comment]
The README already says to run `yarn` which is equivalent to `yarn install`.
